flixel/tile/FlxTilemap.hx

```diff
@@ -48,13 +51,6 @@ class FlxTilemap extends FlxBaseTilemap<FlxTile>
 	 */
 	private static var _helperBuffer:FlxTilemapBuffer = Type.createEmptyInstance(FlxTilemapBuffer);
 	
-	// TODO: remove this hack and add docs about how to avoid tearing problem by preparing assets and some code...
-	/**
-	 * Try to eliminate 1 px gap between tiles in tile render mode by increasing tile scale, 
-	 * so the tile will look one pixel wider than it is.
-	 */
-	public var useScaleHack:Bool = true;
```

Comment:
@bendmorris thank you for comment! 
With new gl renderer it should be possible to avoid this problem with shader based game scaling. But i'll try to reimplement your work for flixel as well.